Web/Presenters/UserPresenter.php

```diff
@@ -125,6 +125,11 @@ function renderEdit(): void
                     $user->setTelegram(empty($this->postParam("telegram")) ? NULL : $this->postParam("telegram"));
                     $user->setCity(empty($this->postParam("city")) ? NULL : $this->postParam("city"));
                     $user->setAddress(empty($this->postParam("address")) ? NULL : $this->postParam("address"));
+                    if(strpos($this->postParam("website") ?? "", "https://") === 0 || strpos($this->postParam("website") ?? "", "http://") === 0) {
+                        $user->setWebsite(empty($this->postParam("website")) ? NULL : $this->postParam("website"));
+                    } else {
+                        $user->setWebsite(empty($this->postParam("website")) ? NULL : "http://" . $this->postParam("website"));
```

Comment:
Parse the protocol (secure or not) in dedicated variable

Reply:
You mean I need to do this:
```php
$website = $this->postParam("website") ?? "";
if(strpos($website, "https://") === 0 || strpos($website, "http://") === 0) {
  $user->setWebsite(empty($this->postParam("website")) ? NULL : $this->postParam("website"));
} else {
  $user->setWebsite(empty($this->postParam("website")) ? NULL : "http://" . $this->postParam("website"));
}
```
?

Reply:
Or even better:
```php
$website = $this->postParam("website") ?? null;
if($website['scheme'] == "https")
    $user->setWebsite($website);
else
    $user->setWebsite("http://".$website);
```

Reply:
Okay

Reply:
Done

Reply:
Oh, I didn't notice the new message

Reply:
But in this case, if you start the domain with "http://", you get this:
![image](https://user-images.githubusercontent.com/50026114/140811203-b6ec6b97-d7c5-4f58-9d4d-bf3cb2c5ed58.png)
For some reason, the code below also has a scenario like this
```php
if($website['scheme'] == "https" || $website['scheme'] == "http")
    $user->setWebsite($website);
else
    $user->setWebsite("http://".$website);
```

Reply:
i forgot about `parse_url()` lol

```php
if(parse_url($website)['scheme'] == "https" || parse_url($website)['scheme'] == "http")
    $user->setWebsite($website);
else
    $user->setWebsite("http://".$website);
```

Reply:
I also liked to add some function like "validate url" like [here](https://www.php.net/manual/en/function.parse-url.php#106731)

Reply:
But the original VK does not check the URL in any way (except for adding http://)